image-customize

```diff
@@ -52,91 +51,91 @@ def prepare_install_image(base_image, install_image, resize):
     return install_image
 
 
-def run_command(machine_instance, commandlist):
-    """Run command inside image"""
-    for foo in commandlist:
+class ActionBase(argparse.Action):
+    '''Keep an ordered list of actions'''
+
+    def __call__(self, parser, namespace, value, option_string=None):
+        namespace.actions.append((self.execute, value))
```

Comment:
Still a bit gross that `actions` is hardcoded here.

Reply:
Oops, thanks, forgot about that. Fixed.